collections/Laud_Misc/MS_Laud_Misc_268.xml

```diff
@@ -30,20 +30,48 @@
                   <institution>University of Oxford</institution>
                   <repository>Bodleian Library</repository>
                   <idno type="shelfmark">MS. Laud Misc. 268</idno>
-                  <idno type="ieArk">ark:29072/x07h149q3994</idno><idno type="crArk">ark:29072/x0jm214p2704</idno><altIdentifier type="internal">
+                  <idno type="ieArk">ark:29072/x07h149q3994</idno>
+                  <idno type="crArk">ark:29072/x0jm214p2704</idno>
+                  <altIdentifier type="internal">
                      <idno type="SCN">1229</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>A Bede on Acts; Germany, 14th century. B Johannes Damascenus, <title>De fide orthodoxa</title>; Italy (?), late 13th / early 14th century. C Johannes Damascenus, Ps.-Augustine; France (?Paris), 14th century, first half</head>
                <physDesc>
-                  <p n="composite">Composite:</p>
+                  <p n="composite">Composite: three manuscripts probably brought together at Eberbach Abbey around the mid-fourteenth century..</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <extent>i + 120 + i leaves (flyleaves early modern parchment)
+                           <dimensions unit="mm" type="leaf">
+                              <height>305</height>
+                              <width>210</width>
+                           </dimensions></extent>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding notAfter="1639" notBefore="1638">
+                        <p>Standard binding of the Laudian collection, calf over pasteboards, c. 1638-9; rebacked.</p>
```

Comment:
I often find it helpful to put rebackings in a second `<binding>` element with the date – but perhaps this is unnecessary detail?